src/python_server/server.py

```diff
@@ -0,0 +1,37 @@
+import zmq
+import time
+import sys
+import os
+import capnp
+
+script_dir=os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(0, os.path.join(script_dir, "../"))
+
+from capnp_serial import messages_capnp
+from capnp_serial import mapping_capnp
+
+port = "5556"
+if len(sys.argv) > 1:
+    port =  sys.argv[1]
+
+context = zmq.Context()
+socket = context.socket(zmq.REP)
+socket.bind("tcp://*:%s" % port)
+
+while True:
```

Comment:
Can you change this to functions? Instead of being a global script, such that it is easier to test if this keepsgetting more completcated.